### ISSUE TITLE
Updated @tensorflow/tfjs-node-gpu from 0.1.21 to 0.2.1 got: Cannot read property 'name' of null.

### ISSUE THREAD

**Issue by iacovlev-pavel** [tensorflow/tfjs#1053]
#### TensorFlow.js version
0.2.1

#### Browser version
N/A

#### Describe the problem or feature request
upon running the following code:
```
const model = await tf.loadFrozenModel(modelPath, weightsPath);
const shape = [1, 2560, 1920, 3];
const tensor = tf.fill(shape, 0, 'int32');
await model.executeAsync(
  { image_tensor: tensor },
  ['detection_boxes', 'detection_scores', 'detection_classes', 'num_detections'],
);
```

I get the following error:
```
error:  message=Cannot read property 'name' of null, stack=TypeError: Cannot read property 'name' of null
    at Engine.runKernel (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-core\dist\engine.js:124:42)
    at Object.cropAndResize_ [as cropAndResize] (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-core\dist\ops\image_ops.js:184:40)
    at Object.exports.executeOp (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-converter\dist\src\operations\executors\image_executor.js:26:31)
    at Object.executeOp (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-converter\dist\src\operations\operation_executor.js:36:26)
    at _loop_1 (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-converter\dist\src\executor\graph_executor.js:301:52)
    at GraphExecutor.processStack (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-converter\dist\src\executor\graph_executor.js:327:13)
    at GraphExecutor.<anonymous> (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-converter\dist\src\executor\graph_executor.js:279:41)
    at step (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-converter\dist\src\executor\graph_executor.js:40:23)
    at Object.next (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-converter\dist\src\executor\graph_executor.js:21:53)
    at fulfilled (D:\Projects\pk-next\node_modules\@tensorflow\tfjs-converter\dist\src\executor\graph_executor.js:12:58)
```

The model is based on `faster_rcnn_inception_v2_coco`

With @tensorflow/tfjs-node-gpu@0.1.21 everything works as expected.

**Comment by bileschi** [comment]
There was an issue not to long ago dealing with underspecification of model formats.  That is my first suspicion.  Can you append your model (no need to append the weights for now, if they are very large.

**Comment by iacovlev-pavel** [comment]
I hope this is what is needed, I did not include the shards since they are 180mb.
[model.zip](https://github.com/tensorflow/tfjs/files/2722928/model.zip)

**Comment by iacovlev-pavel** [comment]
@bileschi bileschi

Here is a more reproducible use case:

Download the model:
http://download.tensorflow.org/models/object_detection/faster_rcnn_resnet101_coco_2018_01_28.tar.gz

Convert the model:
```
tensorflowjs_converter --input_format=tf_saved_model --output_node_names="detection_boxes,detection_scores,num_detections,detection_classes" --saved_model_tags=serve model/faster_rcnn_resnet101_coco_2018_01_28/saved_model model/faster_rcnn_resnet101_coco_2018_01_28/web_model
```

Run the code
```
const model = await tf.loadFrozenModel(modelPath, weightsPath);

const shape = [1, 2560, 1920, 3];
const imageTensor = tf.fill(shape, 0, 'int32');
const predictions = await model.executeAsync(
  { image_tensor: imageTensor },
  ['detection_boxes', 'detection_scores', 'detection_classes', 'num_detections'],
);

imageTensor.dispose();
predictions.forEach(prediction => prediction.dispose());
```

**Comment by iacovlev-pavel** [comment]
Seems to be fixed in 0.3.0
